packages/runtime/src/runtimes/react/hooks/use-resolved-value.ts

```diff
@@ -0,0 +1,8 @@
+import { ControlDefinition, type DataType, type ResolvedValueType } from '@makeswift/controls'
+
+export function useResolvedValue(
+  data: DataType<ControlDefinition> | undefined,
+  definition: ControlDefinition,
+): ResolvedValueType<ControlDefinition> {
+  return definition.resolveValue(data).readStableValue()
```

Comment:
Now that I see this implementation it does seem that there's an assumption that `subscribe` is still not implemented for any of the controls and I'm guessing it will be implemented later?

Reply:
Yep, it's no-op for all "basic" controls and it will remain a no-op. I could have not included it into this iteration of the `Resolvable` interface, but decided to leave it in since it will remain a no-op.

Reply:
This being only present for stuff like swatches seems like accidental complexity from the setup evolving from the old hooks and the concept of an API that also holds the cache for API resources. It would be nice if we had resolved values in state and we could read and subscribe to them and that is consistent whether they're basic values or not. Regardless, that's something that can be explored later